README.md

```diff
@@ -66,30 +66,52 @@ You should see output similar to the following:
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+## Running the App with Gunicorn - HELP! This does not work... how do I use the Ubuntu shell?
+
+Install gunicorn:
+```bash
+$ pip install gunicorn
```

Comment:
Rather than tracking gunicorn with pip, we can track it with poetry (`poetry add gunicorn`) and make it available to our app by default when we run `poetry install`

Reply:
Thanks, done.